### PR TITLE
Turkish translation

### PR DESCRIPTION
**Pull request by cihanyakar**
Turkish translation has been added.

### CODE REVIEW

**Comment by cihanyakar** [comment]
#23

**Comment by sthewissen** [comment]
Awesome! Could I trouble you for 3 more texts that I forgot? 

"Sign in"
"Swipe to continue"
"Try with a demo account"

Feel free to leave the translations here and I'll make sure they make their way in :)

**Comment by cihanyakar** [comment]
Sure, I'll have added tomorrow at the latest.

File: MVP/Resources/Translations.tr.resx [hunk 1]
```diff
@@ -0,0 +1,465 @@
...
+    <value>Emeği Geçenler</value>
+  </data>
+  <data name="acknowledgementtype_tester" xml:space="preserve">
+    <value>Tester</value>
+  </data>
+  <data name="acknowledgementtype_translator" xml:space="preserve">
+    <value>Çevirmen</value>
+  </data>
+  <data name="sign_in" xml:space="preserve">
+    <value>Sign in</value>
```
Comment (sthewissen) [inline MVP/Resources/Translations.tr.resx:457]: @cihanyakar I already added the 3 English versions of the labels here for you :)

Comment (cihanyakar) [inline MVP/Resources/Translations.tr.resx:457]:
```suggestion
    <value>Oturum Aç</value>
```

File: MVP/Resources/Translations.tr.resx [hunk 2]
```diff
@@ -0,0 +1,465 @@
...
+  </data>
+  <data name="swipe_to_continue" xml:space="preserve">
+    <value>Swipe to continue</value>
```
Comment (cihanyakar) [inline MVP/Resources/Translations.tr.resx:460]:
```suggestion
    <value>Devam etmek için kaydırın</value>
```

File: MVP/Resources/Translations.tr.resx [hunk 3]
```diff
@@ -0,0 +1,465 @@
...
+  </data>
+  <data name="try_with_demo_account" xml:space="preserve">
+    <value>Try with a demo account</value>
```
Comment (cihanyakar) [inline MVP/Resources/Translations.tr.resx:463]:
```suggestion
    <value>Bir demo hesap ile deneyin</value>
```

**Review by cihanyakar — COMMENTED** [review]
I think it is OK now

**Review by sthewissen — APPROVED** [review]

**Comment by sthewissen** [comment]
Awesome! I also had someone else respond that he wanted to do Turkish, so I asked him to review this PR. That way at least 2 people have looked at it and we can merge this once the review is done 👍

Thanks for the effort so far!

File: MVP/Resources/Translations.tr.resx [hunk 4]
```diff
@@ -0,0 +1,465 @@
...
+    <value>Lisanslar</value>
+  </data>
+  <data name="about_licenses_description" xml:space="preserve">
+    <value>Bu uygulama devlerin omuzlarında duruyor. Onlar olmadan bu uygulama mümkün olamazdı. Bu olağanüstü kişilikler hakkında daha ayrıntılı bilgileri, bu uygulamada kullanılan harika kütüphanelerin lisansları altında bulabilirsiniz.</value>
+  </data>
+  <data name="view_licenses" xml:space="preserve">
+    <value>Lisansları görüntüle</value>
+  </data>
+  <data name="progress_loading_url_data" xml:space="preserve">
+    <value>Getting URL metadata...</value>
```
Comment (polatpinar) [inline MVP/Resources/Translations.tr.resx:259]: Maybe "URL meta verisi alınıyor"?

**Review by polatpinar — CHANGES_REQUESTED** [review]
Looks good

**Review by polatpinar — APPROVED** [review]

**Pull request merged**